### PR TITLE
feat: add new javascript quiz

### PR DESCRIPTION
**Pull request by Evavic44**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1528,6 +1528,17 @@ const javascriptQuiz = [
       "The toFixed() method converts a number to a string and rounds it to a specified number of decimals.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed",
   },
+  {
+    Question:
+      "In JavaScript, what operator has the highest precedence among these options",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1533]:
```suggestion
      "In JavaScript, which operator has the highest precedence among these options?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by Evavic44** [comment]
> Congrats on another contribution to our repository. 🎉
> 
> We look forward to future contributions. 👍

Thanks a lot, @jdwilkin4

**Pull request merged**